modules/creating-an-accelerator-profile.adoc

```diff
@@ -0,0 +1,35 @@
+:_module-type: PROCEDURE
+
+[id="creating-an-accelerator-profile_{context}"]
+= Creating an accelerator profile
+
+[role='_abstract']
+To configure accelerators for your data scientists to use in {productname-short}, you must create an associated accelerator profile. 
+
+.Prerequisites
+* You have logged in to {openshift-platform}.
+* You have the `cluster-admin` role in {openshift-platform}.
+* The `acceleratorprofiles.dashboard.opendatahub.io` custom resource definition (CRD) exists in your deployment.
+* You are aware of the correct namespace to create the accelerator profile in. 
+
+.Procedure
+. In the {openshift-platform} web console, in the *Administrator* perspective, click *Administration* -> *CustomResourceDefinitions*.
+. Click the `acceleratorprofiles.dashboard.opendatahub.io` custom resource definition (CRD).
++
+A details page for the custom resource definition (CRD) opens.
+. Click the *Instances* tab.
+. Click *Create AcceleratorProfile*. 
++ 
+The *Create AcceleratorProfile* page opens and displays an embedded YAML editor.
+. Enter or paste the YAML code directly in the embedded editor.
+. Click *Create*.
+
+.Verification
+* The *Accelerator* list displays on the *Start a notebook server* page. After you select an accelerator, the *Number of accelerators* field appears allowing you to choose the number of accelerators for your notebook server.  
```

Comment:
- Same as above RE: 'displays' > 'appears'.
- Could avoid use of 'allowing you' by rephrasing slightly to something like:
'After you select an accelerator, the *Number of accelerators* field appears, which you can use to choose the number of accelerators for your notebook server.'

Reply:
Changed.